crates/debugger_ui/src/debugger_panel.rs

```diff
@@ -618,7 +627,25 @@ impl DebugPanel {
         event: &OutputEvent,
         cx: &mut ViewContext<Self>,
     ) {
-        cx.emit(DebugPanelEvent::Output((*client_id, event.clone())));
+        let existing_item = self
+            .pane
+            .read(cx)
+            .items()
+            .filter_map(|item| item.downcast::<DebugPanelItem>())
+            .any(|item| {
+                let item = item.read(cx);
+
+                item.client_id() == *client_id
+            });
+
+        if existing_item {
```

Comment:
Can we pls just always send the event and push the message queue? And when the client has stopped (shutdown) we should remove the queue again. Because you could have multiple debug sessions and each debug session should show the same output log

Reply:
Yes! I'll make the change.